CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 - Deprecate the old `StringKey.discardAlertCancel` and `StringKey.discardAlertAction` string keys for overriding the discard alert buttons as they had incosistent behavior between iOS and Android ([#1001](https://github.com/Instabug/Instabug-React-Native/pull/1001)).
 - Deprecate the old `StringKey.reproStepsListItemNumberingTitle` string key for overriding the repro steps list item (screen) title as it had incosistent behavior between iOS and Android ([#1002](https://github.com/Instabug/Instabug-React-Native/pull/1002)).
 
+### Changed
+
+- Bump Instabug iOS SDK to 11.14.0 ([#1020](https://github.com/Instabug/Instabug-React-Native/pull/1020)) [See release notes](https://github.com/Instabug/Instabug-iOS/releases/tag/11.14.0)
+
```

Comment:
Let's move the "Changed" section right after "Added" as our Changelog Guidelines document states.

Also, I have a small suggestion to align our Changelog formatting 🙏🏼

```suggestion
### Changed

- Bump Instabug iOS SDK to v11.14.0 ([#1020](https://github.com/Instabug/Instabug-React-Native/pull/1020)). [See release notes](https://github.com/Instabug/Instabug-iOS/releases/tag/11.14.0).
```

Reply:
Thanks for clarifying and catching this 💪 . I've updated it, please recheck.